site/_posts/2020-11-13-securing-flutter-oauth.md

```diff
@@ -0,0 +1,766 @@
+
+Next, we move on to setting up the Flutter app project.
+
+## Setting up the Flutter project
+
+Setting up the Flutter project is simple and easy for every OS. We can follow the instructions in the [Flutter documentation](https://flutter.dev/docs/get-started/install). But to make things easier, a quick outline of the setup steps is provided below.
+
+First, we need to clone the Flutter SDK from GitHub by running the following command in our device terminal:
+
+```shell
+git clone https://github.com/flutter/flutter.git
+```
+
+Note that we need to choose an appropriate directory in which to clone the SDK.
+
+Next, we need to add the flutter tool to our path by running the following command in the terminal (and if you are doing this more than once, perhaps to your `.bashrc` or `.zshrc`):
+
+```shell
+export PATH="$PATH:`pwd`/flutter/bin"
+```
+
+We can check if everything is configured properly by running the following command in our terminal window:
+
+```shell
+flutter doctor
+```
+
+If everything is configured properly, we will see the following result in our terminal window:
+
+{% include _image.liquid src="/assets/img/blogs/securing-flutter-app/flutter-doctor-output.png" alt="Output of the flutter doctor command." class="img-fluid" figure=false %}
+
+If you didn't see the above messages, please review the [more detailed installation instructions](https://flutter.dev/docs/get-started/install) on the Flutter site.
+
+Note that Flutter depends on iOS and/or Android development environments being installed. Doing so is beyond the scope of this tutorial, however. If you need to do that, please consult the platform installation instructions for Android or iOS as appropriate.
+
+## Creating a new Flutter app
+
+In order to create and set up the new flutter app, run the following command. You'll want to be in a directory where the project should live.
+
+
+```shell
+flutter create fusionauth_demo
+```
+
+After the installation process completes, we will see that the `fusionauth_demo` directory contains all the flutter starter app configuration. We can open the project directory with VScode by running the following command:
+
+```shell
+code fusionauth_demo
+```
+
+If you don't have VSCode, open the files in whatever text editor you'd prefer; we're not picky!
+
+We can run our new project in the actual device or an emulator to confirm everything is working before we customize any code. Do so by running the following command in the project directory:
+
+
+```shell
+flutter run
+```
+
+After running this, we will get the list of emulators or devices in which we want to run the project. We can simply choose the one in which we want our code to run.
+
+{% include _image.liquid src="/assets/img/blogs/securing-flutter-app/flutter-choose-device.png" alt="Output of the flutter run command." class="img-fluid" figure=false %}
```

Comment:
This terminal output image is really hard to read. His terminal is setup with a pretty non-standard color scheme. It would be great if he could use a standard black background with white and bright colored text. It might also be nice to calm down the prompt. Having the date and time on the right of the prompt is pretty confusing.

Reply:
Will ask if he can re-screenshot that.